src/data/privacy_de.json

```diff
@@ -40,7 +41,7 @@
                 "<ul><li><a href='/assets/documents/cwa-privacy-notice-1.5-de.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in Deutsch</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.5-en.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in English</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.5-tr.pdf'>Datenschutzerklärung Version 1.5 bis 1.7.1 in Türkisch</a></li></ul>",
                 "<ul><li><a href='/assets/documents/cwa-privacy-notice-1.3-de.pdf'>Datenschutzerklärung bis Version 1.3 in Deutsch</a></li><li><a href='/assets/documents/cwa-privacy-notice-1.3-en.pdf'>Datenschutzerklärung bis Version 1.3 in English</a></li></ul>"
             ],
-            "textcaption": "Letzte Aktualisierung: 16. April 2021"
+            "textcaption": "Letzte Aktualisierung: 1. May 2021"
```

Comment:
1. Mai 2021

Reply:
This is auto generated and should not be changed.

Reply:
See https://github.com/corona-warn-app/cwa-website/blob/ae7c0452903a659cdfd2dc3c7194db686cf9acc6/gulpfile.js#L333

Reply:
2. may

Reply:
ok